core/src/main/java/org/apache/accumulo/testing/core/continuous/ContinuousIngest.java

```diff
@@ -45,11 +45,43 @@
   private static final byte[] EMPTY_BYTES = new byte[0];
 
   private static List<ColumnVisibility> visibilities;
+  private static long lastPause;
+  private static long pauseWaitMs;
 
   private static ColumnVisibility getVisibility(Random rand) {
     return visibilities.get(rand.nextInt(visibilities.size()));
   }
 
+  private static boolean pauseEnabled(Properties props) {
+    String value = props.getProperty(TestProps.CI_INGEST_PAUSE_ENABLED);
+    return value != null && value.equals("true");
```

Comment:
Could use `return Boolean.parseBoolean(value);` here.

Reply:
Fixed in 84fcd3a630